[bitnami/zookeeper] The production's configuration can not work for me

**Which chart**:
bitbnami/zookeeper  zookeeper-5.21.0.tgz

**Describe the bug**
[ERROR] templates/: template: zookeeper/templates/statefulset.yaml:135:31: executing "zookeeper/templates/statefulset.yaml" at <.Values.autopurge.purgeInterval>: nil pointer evaluating interface {}.purgeInterval

Error: 1 chart(s) linted, 1 chart(s) failed

**To Reproduce**
Steps to reproduce the behavior:
1. helm pull bitbnami/zookeeper
2. tar -xvxf zookeeper-5.21.0.tgz
3. cd zookeeper
// I'd like to overwrite the `values.yaml`  use the `values-production.yaml`
4. cp values-production.yaml values.yaml
5.Use the command  with `helm lint .` in order to check the charts

**Expected behavior**
[ERROR] templates/: template: zookeeper/templates/statefulset.yaml:135:31: executing "zookeeper/templates/statefulset.yaml" at <.Values.autopurge.purgeInterval>: nil pointer evaluating interface {}.purgeInterval


**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"clean", GoVersion:"go1.13.12"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.5", GitCommit:"e6503f8d8f769ace2f338794c914a96fc335df0f", GitTreeState:"clean", BuildDate:"2020-06-26T03:47:41Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.6", GitCommit:"d32e40e20d167e103faf894261614c5b45c44198", GitTreeState:"clean", BuildDate:"2020-05-20T13:08:34Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
Add any other context about the problem here.

--- a/bitnami/zookeeper/Chart.yaml
+++ b/bitnami/zookeeper/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 5.21.1
+version: 5.21.2
 appVersion: 3.6.1
 description: A centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services for distributed applications.
 keywords:

--- a/bitnami/zookeeper/values-production.yaml
+++ b/bitnami/zookeeper/values-production.yaml
@@ -148,6 +148,14 @@ listenOnAllIPs: false
 ##
 allowAnonymousLogin: true
 
+autopurge:
+  ## Retains the snapRetainCount most recent snapshots and the corresponding transaction logs and deletes the rest
+  ##
+  snapRetainCount: 3
+  ## The time interval in hours for which the purge task has to be triggered. Set to a positive integer (1 and above) to enable the auto purging.
+  ##
+  purgeInterval: 0
+
 auth:
   ## Use existing secret (ignores previous password)
   ##
@@ -176,6 +184,8 @@ heapSize: 1024
 ##
 logLevel: ERROR
 
+## Data log directory. Specifying this option will direct zookeeper to write the transaction log to the dataLogDir rather than the dataDir.
+## This allows a dedicated log device to be used, and helps avoid competition between logging and snaphots.
 ## Example:
 ## dataLogDir: /bitnami/zookeeper/dataLog
 ##
@@ -418,4 +428,3 @@ metrics:
     #    for: 5m
     #    labels:
     #      severity: critical
-

